core/script_language.h

```diff
@@ -119,7 +119,7 @@ class Script : public Resource {
 	virtual void get_script_method_list(List<MethodInfo> *p_list) const = 0;
 	virtual void get_script_property_list(List<PropertyInfo> *p_list) const = 0;
 
-	virtual int get_member_line(const StringName &p_member) const { return 0; }
+	virtual int get_member_line(const StringName &p_member) const { return -1; }
```

Comment:
I thought it was fine because `GDScript::get_member_line` already [returned -1](https://github.com/godotengine/godot/blob/df61dc4b2bd54a5a40c515493c76f5a458e5b541/modules/gdscript/gd_script.h#L198).